public/index.html

```diff
@@ -88,7 +58,29 @@ <h1 id="title">LaTeX Online</h1>
 <a href="https://github.com/aslushnikov/latex-online#ccommand-line-interface">Tell
 me more</a>
                         </li>
+                    </ol>                    
+                    <h3>Parameters</h3>
+                    <ol>
+                        <li><code>&command=</code> xelatex or pdflatex</li>
```

Comment:
since it's unclear what is this about (huh? Parameters?..), let's rather have a link to the documentation here